jenkins.py

```diff
@@ -36,9 +36,12 @@ def get_updated_tools(repos):
         tags = {str(x).split('--')[1] for x in tags if '--' in x}
         # Fetch last commit hash for tool using `git log`
         p = subprocess.Popen(['git', 'log', '--pretty=oneline', '-n', '1', '--', tool], stdout=subprocess.PIPE)
-        commit, comment = p.stdout.read().split(" ", 1)
-        if commit not in tags:
-            updated_tools.add(tool)
+        try:
+            commit, comment = p.stdout.read().split(" ", 1)
+            if commit not in tags:
+                updated_tools.add(tool)
+        except ValueError as e:
+            pass
```

Comment:
This is lazy. Isn't the problem that `tool`'s value doesn't refer to an existing directory, causing `git` to either produce empty output or an error? IOW, the condition you are trying to prevent can be caught earlier, and with more precision. This current approach just introduces the chance of hiding other problems.